### PR TITLE
Add new logic to re-import orders with errors

### PR DESCRIPTION
**Pull request by mstrzyzewski**
-Create console commands for importing orders with errors and for importing order by its ID
-Add new table in database to store information about orders with errors
-Create admin grid for listing orders with errors
-Create new admin system message for informing about orders that have not been imported

### CODE REVIEW

**Comment by tmotyl** [comment]
Add declare strict type to all new files plz

File: Controller/Adminhtml/Orders/Import.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<?php
+
```
Comment (tmotyl) [inline Controller/Adminhtml/Orders/Import.php]: missing strict

File: Controller/Adminhtml/Orders/Index.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?php
+
```
Comment (tmotyl) [inline Controller/Adminhtml/Orders/Index.php]: same here

File: Model/System/Message/AllegroConnectionStatusMessage.php [hunk 3]
```diff
@@ -3,13 +3,12 @@
...
 
 use Macopedia\Allegro\Model\Api\ClientException;
-use Magento\Framework\Exception\LocalizedException;
 use Magento\Framework\Notification\MessageInterface;
 use Macopedia\Allegro\Model\Api\Credentials;
 use Magento\Framework\App\Config\ScopeConfigInterface;
 
 /**
- * Allegro connection status message class
+ * AllegroConnectionStatusMessage class
```
Comment (tmotyl) [inline Model/System/Message/AllegroConnectionStatusMessage.php]: ? old version was not ideal, but still better than this

File: Model/System/Message/AllegroOrdersWithErrorsMessage.php [hunk 4]
```diff
@@ -0,0 +1,113 @@
+<?php
+
```
Comment (tmotyl) [inline Model/System/Message/AllegroOrdersWithErrorsMessage.php]: same

File: Api/OrderLogRepositoryInterface.php [hunk 5]
```diff
@@ -16,6 +16,12 @@ interface OrderLogRepositoryInterface
      */
     public function getList(SearchCriteriaInterface $searchCriteria) : array;
 
+    /**
+     * @return int
+     * @throws \Exception
+     */
+    public function getCount() : int;
```
Comment (tmotyl) [inline Api/OrderLogRepositoryInterface.php]: this is a breaking change, so we will need to bump version number, which is OK, but please add [!!!] at the beginning of the commit message to make clear this change is breaking

**Comment by mstrzyzewski** [comment]
@tmotyl I pushed fixes as you asked me for. Please let me know if everything's alright now

**Pull request merged**